From a5814f5c13670b83f62365709e47bd5b69010fd8 Mon Sep 17 00:00:00 2001
From: dantonov <dananto02@gmail.com>
Date: Tue, 5 Nov 2024 21:29:38 +0300
Subject: [PATCH 2/2] Added Second change by person3

---
 ConsoleApp1/Program.cs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ConsoleApp1/Program.cs b/ConsoleApp1/Program.cs
index 96504a2..faad273 100644
--- a/ConsoleApp1/Program.cs
+++ b/ConsoleApp1/Program.cs
@@ -8,6 +8,7 @@
             Console.WriteLine("New Hello World from person1");
 
             Console.WriteLine("First change by person3");
+            Console.WriteLine("Second change by person3");
         }
     }
 }
\ No newline at end of file
-- 
2.39.1.windows.1

